src/components/Forms/UserInfoForm.tsx

```diff
@@ -0,0 +1,251 @@
+'use client';
```

Comment:
On this component, it seems okay, but I'm afraid if we have some with more logic, we shouldn't adopt a different structure folder like, creating a folder for UseInfoForm and inside of it we could have an util folder with the util functions. Let me know what is your thoughts

Reply:
Do you mean with the amount of functions?

It depends on the way you want to approach this. Following other languages like Rust or even old ways of React, a lot of the logic that is intrinsic with the component is within itself.
I do think that repeating this code is less hurtful than trying to make it fit for all possible cases and eventually becomes dead code.

We can discuss this though :)